CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * Bug - Persist container exit code in agent state file [#1125](https://github.com/aws/amazon-ecs-agent/pull/1125)
 * Bug - Fixed a bug where the agent could lose track of running containers when Docker APIs timeout [#1217](https://github.com/aws/amazon-ecs-agent/pull/1217)
 * Bug - Task level memory.use_hierarchy was not being set and memory limits were not being enforced [#1195](https://github.com/aws/amazon-ecs-agent/pull/1195)
+* Bug - Fixed a bug where CPU utilization isn't correctly reported on windows [@bboerst](https://github.com/bboerst) [#1219](https://github.com/aws/amazon-ecs-agent/pull/1219)
```

Comment:
Can you:
* Word-wrap this at 80 characters
* s/windows/Windows/
* Change it to past-tense
?

Thanks!